app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
+  def last_post
```

Comment:
- [ ] **[OPTIONAL]** I think it would be more appropriate to give this a more suitable name like `last_three_posts`